26/documentation.html

```diff
@@ -41,7 +41,7 @@ <h3 class="anchor-heading"><a id="introduction" class="anchor-link"></a><a href=
       <h3 class="anchor-heading"><a id="uses" class="anchor-link"></a><a href="#uses">1.2 Use Cases</a></h3>
       <!--#include virtual="uses.html" -->
       <h3 class="anchor-heading"><a id="quickstart" class="anchor-link"></a><a href="#quickstart">1.3 Quick Start</a></h3>
-      <!--#include virtual="quickstart.html" -->
+      <!--#include virtual="quickstart-zookeeper.html" -->
```

Comment:
Also, you'll want to keep the `<!--#include virtual="quickstart.html" -->` links as well

cf https://github.com/apache/kafka-site/pull/313